.vscode/settings.json

```
@@ -1,7 +1,7 @@
 {
   "gopls": {
     "build.env": {
-      "GOPRIVATE": "github.com/cloudy-sky-software/pulumi-provider-framework"
+      "GOPRIVATE": "github.com/phillarson-xyz/pulumi-provider-framework"
```

Last one of these changes to be reverted.

```suggestion
      "GOPRIVATE": "github.com/cloudy-sky-software/pulumi-provider-framework"
```